### PR TITLE
README.md: Add Gentoo installation documentation

### PR DESCRIPTION
**Pull request by cova-fe**
-  Add Gentoo installation instructions, along with pointers to where to find ebuilds
- Only README.md changes, no code differences.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -172,6 +173,30 @@ Funtoo Linux has an autogenerated lazygit package in [dev-kit](https://github.co
...
+sudo eselect repository enable cova
+sudo emaint sync -r cova
+sudo emerge dev-vcs/lazygit
+```
+
+Or you can download the ebuild and install it manually; please consider the example below just as a suggestion to be adapted to your system.
+
+```sh
+su
+LAZYGIT_VERSION="0.39.2" # Replace with the version you want from cova-overlay
```
Comment (jesseduffield) [inline README.md:192]:
```suggestion
LAZYGIT_VERSION="0.39.3" # Replace with the version you want from cova-overlay
```
May as well set to the latest version as of 6 hours ago :)

Comment (cova-fe) [reply README.md:192]: Fair enough, I update the ebuild but forgot the PR :)

**Comment by jesseduffield** [comment]
LGTM!

**Comment by band-a-prend** [comment]
Btw `dev-vcs/lazygit` is also available in [Gentoo overlay Guru](https://github.com/gentoo/guru) repository (maintained collaboratively by Gentoo users) since 2020. This repo is tracked by Repology.

But I still didn't add information about it here :(  
My fault.

**Pull request merged**